src/components/admin/adminUserReport/AdminReportDetail.styled.ts

```
@@ -0,0 +1,144 @@
+      height: 1px;
+      top: 78%;
+      left: -11rem;
+      background-color: ${({ theme }) => theme.color.primary};
+      transform-origin: 0 100%;
+      transform: rotate(0deg);
+
+    }
+  }
+`;
+
+export const Date = styled.p`
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)